### PR TITLE
Add the android gitignore to sample app

### PR DESCRIPTION
**Pull request by jhen0409**

### CODE REVIEW

File: Examples/SampleApp/.gitignore [hunk 1]
```diff
@@ -21,6 +21,12 @@ DerivedData
 *.ipa
 *.xcuserstate
 
+# Android/IJ
+.idea
```
Comment (ide) [inline Examples/SampleApp/.gitignore]: Latest Android Studio uses this instead:
```
*/.idea/workspace.xml
*/.idea/libraries
```

**Comment by kmagiera** [comment]
Is this change ignoring anything that is not already covered by .gitignore in the root directory https://github.com/facebook/react-native/blob/master/.gitignore ?

**Comment by ide** [comment]
Right.. this should go under _gitignore instead.

**Comment by jhen0409** [comment]
Oh, sorry. I'm missed _gitignore file. :|

**Comment by mkonicek** [comment]
@jhen0409  Do you want to update this PR or send a new one?

**Comment by jhen0409** [comment]
@mkonicek I send the new PR #2776. Thx :)

**Pull request closed without merging**